### ISSUE TITLE
Any type in OpenApi

### ISSUE THREAD

**Issue by dennis-wey** [koxudaxi/datamodel-code-generator#205]
**Describe the bug**
Loading a valid OpenAPI spec file from fastApi using Pydantic Classes which includes Any result in error.

**To Reproduce**

Minimal Example fastApi application:
from typing import Any

```
from fastapi import FastAPI
from pydantic import BaseModel

app = FastAPI()


class Item(BaseModel):
    foo: str
    bar: Any


@app.post("/")
def read_root(item: Item):
    return {item.foo: item.bar}
```

Reurn this openapi specification (I removed some error responses and turn it to yaml but according to swagger editors this is a valid file):

```
info:
  title: FastAPI
  version: 0.1.0
openapi: 3.0.2
paths:
  /:
    post:
      operationId: read_root__post
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/Item'
        required: true
      responses:
        '200':
          content:
            application/json:
              schema: {}
          description: Successful Response
      summary: Read Root
components:
  schemas:
    Item:
      properties:
        bar:
          title: Bar
        foo:
          title: Foo
          type: string
      required:
      - foo
      title: Item
      type: object
```

Running now 
```
$ datamodel-codegen --input openapi.yml --output test.py
```

Will give me the error:
```
Traceback (most recent call last):
  File "/home/docker/venv3.6/lib/python3.6/site-packages/datamodel_code_generator/__main__.py", line 230, in main
    aliases=aliases,
  File "/home/docker/venv3.6/lib/python3.6/site-packages/datamodel_code_generator/__init__.py", line 180, in generate
    result = parser.parse()
  File "/home/docker/venv3.6/lib/python3.6/site-packages/datamodel_code_generator/parser/base.py", line 339, in parse
    self.parse_raw()
  File "/home/docker/venv3.6/lib/python3.6/site-packages/datamodel_code_generator/parser/openapi.py", line 25, in parse_raw
    self.parse_raw_obj(obj_name, raw_obj, ['#/components', 'schemas', obj_name])
  File "/home/docker/venv3.6/lib/python3.6/site-packages/datamodel_code_generator/parser/jsonschema.py", line 628, in parse_raw_obj
    self.parse_object(name, obj, path)
  File "/home/docker/venv3.6/lib/python3.6/site-packages/datamodel_code_generator/parser/jsonschema.py", line 391, in parse_object
    fields = self.parse_object_fields(obj, path)
  File "/home/docker/venv3.6/lib/python3.6/site-packages/datamodel_code_generator/parser/jsonschema.py", line 359, in parse_object_fields
    field_types = self.get_data_type(field)
  File "/home/docker/venv3.6/lib/python3.6/site-packages/datamodel_code_generator/parser/jsonschema.py", line 169, in get_data_type
    raise ValueError(f'invalid schema object {obj}')
ValueError: invalid schema object items=None uniqueItem=None type=None format=None pattern=None minLength=None maxLength=None minimum=None maximum=None multipleOf=None exclusiveMaximum=None exclusiveMinimum=None additionalProperties=None oneOf=[] anyOf=[] allOf=[] enum=[] writeOnly=None properties=None required=[] ref=None nullable=False x_enum_varnames=[] description=None title='Bar' example=None examples=None default=None
```

**Expected behavior**
The command should run through successfully and the `bar` property should be created as `Any` type.

**Version:**
 - OS: Debian GNU/Linux 9
 - Python Verison: 3.6.6
 - datamodel-code-generator Version 0.5.29

**Comment by koxudaxi** [comment]
@dennis-wey 
Thank you for creating this issue.

I have confirmed the problem. 
I will fix it.

**Comment by koxudaxi** [comment]
@dennis-wey 
I fixed it.
I have released a new version as `0.5.30`
